ui/imports/shared/controls/chat/MessageReactionsRow.qml

```diff
@@ -1,32 +1,98 @@
 import QtQuick
 
+import StatusQ
 import StatusQ.Core.Theme
+import StatusQ.Controls
 
 import shared.controls.chat
 
+import SortFilterProxyModel
+
 Row {
     id: root
 
-    property var defaultEmojiReactionsModel
+    required property StatusEmojiModel emojiModel
```

Comment:
It would be nice to describe which are the properties expected for this specific model